src/Microsoft.ML.Transforms/Text/CharTokenizeTransform.cs

```diff
@@ -5,28 +5,37 @@
 #pragma warning disable 420 // volatile with Interlocked.CompareExchange
```

Comment:
I know you didnt change this - but just curious, why do we have variables set as volatile when we use Interlocked operations?